tests/GetOptionKit/ValueTypeTest.php

```diff
@@ -74,5 +76,17 @@ public function testEmailType()
         ok( $email->test('test@gmail.com') );
         $this->assertFalse($email->test('test@test'));
     }
+
+    public function testRegexType()
+    {
+        $regex = new RegexType;
+        $regex->option = '#^Test$#';
+        ok( $regex->test('Test') );
+        $this->assertFalse($regex->test('test'));
+
+        $regex->option = '/^([a-z]+)$/';
```

Comment:
I can create a new test file called RegexValueTypeTest.php and move stuff there if its what you mean.